inspire/dojson/hep/schemas/hep-0.0.1.json

```diff
@@ -530,26 +528,32 @@
             },
             "type": "array"
         },
-        "experiment": {
-            "required": [
-                "value"
-            ],
-            "type": "object",
-            "properties": {
-                "curated_relation": {
-                    "default": false,
-                    "type": "boolean",
-                    "description": "Was the experiment actually proofchecked by a cataloguer?"
-                },
-                "recid": {
-                    "type": "integer",
-                    "description": "Record ID of the corresponding experiment."
-                },
-                "value": {
-                    "type": "string",
-                    "description": "Raw string. Do we actually need it at all? Can it be taken from the record ID? NOTE: I am not adding accelerator field. Do we have papers on accelerators but not on experiments? Can we always derive the accelerator from the experiment?"
+        "accelerator_experiments": {
+            "uniqueItems": true,
+            "items": {
+                "type": "object",
+                "properties": {
+                    "curated_relation": {
+                        "default": false,
+                        "type": "boolean",
+                        "description": "Was the experiment actually proofchecked by a cataloguer?"
+                    },
+                    "recid": {
+                        "type": "integer",
+                        "description": "Record ID of the corresponding experiment."
+                    },
+                    "experiment": {
```

Comment:
You should consider the 3 component `experiment`, `accelerator` and `institution` as the raw strings, that are then used to build the link. Then upon indexing we can fetch these values from the linked record.